packages/dom/src/dom/is-text-field.js

```diff
@@ -1,15 +1,20 @@
+/* eslint-disable jsdoc/valid-types */
```

Comment:
What's this for?

Reply:
Whoops, this was left over from when I tried to make this a type guard, but it can be removed now. For the record, the type guard syntax sets off `jsdoc/valid-types`:

```
/**
 * @param {Element} element
 * @return {element is HTMLInputElement | HTMLTextAreaElement}
 */
```

We frequently have to disable `jsdoc/valid-types` to be able to use valid TypeScript syntax in the curly braces.